### ISSUE TITLE
Error with Node.js v18 due to MD5 usage

### ISSUE THREAD

**Issue by tamoreton** [kentaroi/eleventy-plugin-rev#2]
I ran into an error while using `eleventy-plugin-rev` with Node.js version 18. The error is: `error:0308010C:digital envelope routines::unsupported`. It appears to arise from the use of the `crypto.createHash('md5')` method in the plugin, as seen here: https://github.com/kentaroi/eleventy-plugin-rev/blob/50a5cc67d570301ddbdd7eaf9eb82a0ff3cef6d9/lib/eleventy-plugin-rev.js#L10.

MD5 is deprecated in OpenSSL 3.0 as per: https://www.openssl.org/docs/man3.0/man3/MD5.html. This seems to be the cause of the error when using the plugin with Node.js v18, which likely uses OpenSSL 3.0 or later.

Possible solutions might include updating the plugin to use a different hashing algorithm, like SHA-256, which is not deprecated.

Thank you for looking into this.

**Comment by kentaroi** [comment]
Thanks for filing the issue, and sorry for the late reply.
I cannot reproduce it in my local environment and no errors were seen in [CI](https://github.com/kentaroi/eleventy-plugin-rev/actions/runs/6902291512).

However, according to the documentation of Node:

> The algorithm is dependent on the available algorithms supported by the version of OpenSSL on the platform.

The issue you have detailed must certainly exist.

I'll fix it, but to work around it, you can use your preferred algorithm as follows:

```node
const rev = require('eleventy-plugin-rev');
rev.settings.hashAlgorithm = 'sha256';

module.exports = function(eleventyConfig) {
  eleventyConfig.addPlugin(rev);
  // ...
};
```

**Comment by tamoreton** [comment]
That solves the issue, thanks!

**Comment by kentaroi** [comment]
I have just published v2.0.0, which changes the default hash algorithm.
